### PR TITLE
Add function `to_felt_or_relocatable`

### PR DESCRIPTION
**Pull request by Juan-M-V**
Add function `to_felt_or_relocatable` to globals in `PyVm::execute_hint`
Add unit tests
Uncomment `is_zero` python test

### CODE REVIEW

File: src/vm_core.rs [hunk 1]
```diff
@@ -22,14 +23,15 @@ use std::any::Any;
 use std::collections::HashMap;
 use std::{cell::RefCell, rc::Rc};
 
-const GLOBAL_NAMES: [&str; 16] = [
+const GLOBAL_NAMES: [&str; 17] = [
```
Comment (Oppen) [inline src/vm_core.rs:26]: Do we need to specify the size explicitly here?

**Review by Oppen — CHANGES_REQUESTED** [review]
Overall a very nice PR. We're missing a test for the relocatable case and we're good to go.

File: src/vm_core.rs (hunk 1 above)
Comment (fmoletta) [reply src/vm_core.rs:26]: The size is needed in order to fulfill the `Sized` trait requirement

**Comment by fmoletta** [comment]
> Overall a very nice PR. We're missing a test for the relocatable case and we're good to go.

Added test for relocatable test!

**Review by Oppen — APPROVED** [review]

File: hints_tests.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def test_program(program_name: str):
     test_program("assert_lt_felt")
     test_program("assert_not_equal")
     test_program("reduce_and_nondet_bigint3")
-    test_program("is_zero")
+    # test_program("is_zero") # Custom Hint Error: NameError: name 'to_felt_or_relocatable' is not defined
```
Comment (Oppen) [inline hints_tests.py:26]: This should be uncommented, isn't the point of the PR to fix this test?

Comment (fmoletta) [reply hints_tests.py:26]: Yes, must have overided it when updating the branch

**Review by entropidelic — APPROVED** [review]

**Pull request merged**